lib/features/thread/presentation/thread_controller.dart

```diff
@@ -217,8 +227,8 @@ class ThreadController extends BaseController with EmailActionController {
   }
 
   void _resetLoadingMore() {
-    if (loadingMoreStatus == LoadingMoreStatus.running) {
-      loadingMoreStatus = LoadingMoreStatus.idle;
+    if (loadingMoreStatus.value == LoadingMoreStatus.running) {
+      loadingMoreStatus.value = LoadingMoreStatus.idle;
```

Comment:
in here ? should we `canLoadMore = false` also, because, user may scroll multiple times ?

Reply:
No, We have updated `loadingMoreStatus = running` every time we call the api. And update it to `loadingMoreStatus = completed` or `loadingMoreStatus = idle` when the process is completed.